src/vm/methodtable.cpp

```diff
@@ -805,8 +805,8 @@ PTR_MethodTable InterfaceInfo_t::GetApproxMethodTable(Module * pContainingModule
     {
         GCStress<cfg_any>::MaybeTrigger();
 
-        // Make call to obj.GetImplType(interfaceTypeObj)
-        MethodDesc *pGetImplTypeMD = pServerMT->GetMethodDescForInterfaceMethod(MscorlibBinder::GetMethod(METHOD__ICASTABLE__GETIMPLTYPE));
+        // Make call to ICastableHelpers.GetImplType(obj, interfaceTypeObj)
+        MethodDesc *pGetImplTypeMD = MscorlibBinder::GetMethod(METHOD__ICASTABLEHELPERS__GETIMPLTYPE);
```

Comment:
It should be just `PREPARE_NONVIRTUAL_CALLSITE(METHOD__ICASTABLEHELPERS__GETIMPLTYPE)`